### PR TITLE
Feature/style to redlines

### LINKED ISSUE #29: Style the picker according to redlines

### PR DESCRIPTION
**Pull request by mttwc**
Fixes #29 

One thing to note is that when declaring css files in html, the last one wins. So if we want to override office ui fabric, we need to declare our css after theirs.

### CODE REVIEW

File: src/oneNotePicker.scss [hunk 1]
```diff
@@ -2,23 +2,45 @@
...
 #oneNotePicker {
   background: white;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
 }
 
 .menu-list li ul {
+  margin-top: 0px;
```
Comment (anbiniyar) [inline src/oneNotePicker.scss]: if `margin-left` is also 0 then use `margin: 0`

File: src/oneNotePicker.scss [hunk 2]
```diff
@@ -2,23 +2,45 @@
 
 #oneNotePicker {
   background: white;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
 }
 
 .menu-list li ul {
```
Comment (anbiniyar) [inline src/oneNotePicker.scss:11]: if `margin-left` is also `0`, then use `margin: 0`

File: src/oneNotePicker.scss [hunk 3]
```diff
@@ -2,23 +2,45 @@
...
+  display: inline-block;
+  vertical-align: top;
+}
+
+.menu-list label {
+  color: black;
 }
 
 .picker-list-header {
+  margin-top: 0px;
```
Comment (anbiniyar) [inline src/oneNotePicker.scss:33]: you might just want to use `margin: 0`

**Review by anbiniyar — APPROVED** [review]
Minor changes. LGTM

File: src/oneNotePicker.scss (hunk 2 above)
Comment (mttwc) [reply src/oneNotePicker.scss:11]: This is defined in office ui fabric

File: src/oneNotePicker.scss (hunk 3 above)
Comment (mttwc) [reply src/oneNotePicker.scss:33]: This is defined in office ui fabric

**Pull request merged**